woocommerce/woocommerce-gutenberg-products-block
# Make Featured Category centered horizontally

## What

Make Featured Category block centered horizontally. Issue was discovered on Twenty Twenty theme.

Fixes https://github.com/woocommerce/woocommerce-blocks/issues/11868

## Why

The block was not centered when used in posts and was always left-aligned. There's no block option to align it so it should be centered by default.

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

0. Switch to Twenty Twenty theme
1. Create new post
2. Add some block for reference, for example Products (Beta) block
3. Add Featured Category
4. Save and go to frontend
5. Both of the blocks are centered
6. Switch to other themes like Twenty Twenty-One, Twenty Twenty-Two, Twenty Twenty-Four and make sure it behaves the same.


## Screenshots or screencast



| Before | After |
| ------ | ----- |
|   <img width="1400" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/20098064/cb35f55c-b8ff-4ffb-ba87-c51ecba33500">     |     <img width="1408" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/20098064/abca718f-5562-4f7d-8e21-e05001ca7fc2">  |

## WooCommerce Visibility


Required:


## Checklist

Required:


Conditional:


## Changelog


> Featured Category: center the block horizontally
